service/validation/userValidation.js

```diff
@@ -0,0 +1,29 @@
+const Joi = require("joi");
+
+const newUserAuthSchema = Joi.object({
```

Comment:
Prostsze nazwy lepiej, np 
```suggestion
const userSchema = Joi.object({
```